proto/internal/buf.yaml

```diff
@@ -12,6 +12,9 @@ breaking:
   # Uncomment this to temporarily ignore specific files or directories:
   ignore:
     - temporal/server/api/historyservice/v1/
+    - temporal/server/api/deployment/v1/message.proto
+    - temporal/server/api/matchingservice/v1/request_response.proto
+    - temporal/server/api/persistence/v1/task_queues.proto
```

Comment:
do we want to take these out so that we make sure we're not making breaking changes to persistance in main?

Reply:
I checked our changes and they're all additive, so not a big deal if we leave this in. But should probably take it out so we stay protected